engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/details/views/errata-details.html

```diff
@@ -41,6 +41,12 @@ <h2 class="fl">{{ errata.title }}</h2>
           </span>
         </a>
       </li>
+      <li ng-class="{active: stateIncludes('errata.tasks')}">
+        <a translate
+           ui-sref="errata.tasks.index({errataId: errata.id})">
+          Tasks
+        </a>
```

Comment:
I don't think this tab is ever going to actually show anything as the tasks aren't tying themselves to the errata.  We might can do that, but its not currently being done.